config/reports.json

```diff
@@ -428,6 +430,132 @@
 			"timeseries": {
 				"enabled": false
 			}
+		},
+		"workboxAdoption": {
+			"name": "Workbox Adoption",
+			"type": "URLs",
+			"downIsBad": true,
+			"description": "[Workbox](https://developers.google.com/web/tools/workbox/) is a set of libraries and Node.js modules that make it easy to cache assets and take full advantage of features used to build Progressive Web Apps. This report shows the number of PWA URLs that use them.",
+			"histogram": {
+				"minDate": "2016_01_01"
```

Comment:
I'm not seeing any histogram queries in the other PR. If that's the case, you don't need to set a minDate for the histograms and you should disable them entirely with 

```
 			"histogram": {
 				"enabled": false
 			}
```